data/h3_raster_importer/Makefile

```diff
@@ -162,12 +162,13 @@ download-pasture:
 	wget -q -O $(WORKDIR_GLW3)/pasture/harvestArea/CroplandPastureArea2000_Geotiff.zip https://s3.us-east-2.amazonaws.com/earthstatdata/CroplandPastureArea2000_Geotiff.zip
 
 #unzip
+#remove no data values
 extract-pasture: download-pasture
 	unzip -j -o -u $(WORKDIR_GLW3)/pasture/harvestArea/CroplandPastureArea2000_Geotiff.zip */Pasture2000_5m.tif -d $(WORKDIR_GLW3)/pasture/harvestArea
-	gdal_translate -a_nodata 0 -of GTiff $(WORKDIR_GLW3)/pasture/harvestArea/Pasture2000_5m.tif \
+	gdal_translate -a_nodata -3.402823e+38 -of GTiff $(WORKDIR_GLW3)/pasture/harvestArea/Pasture2000_5m.tif \
```

Comment:
why is the nodata value this big? 
thinking out loud: is it safe for and always the the same value or can it depend on the platform/implementation precision?